stumpy/aamp.py

```diff
@@ -95,12 +99,14 @@ def _compute_diagonal(
 
         for i in iter_range:
             if i == 0 or (k < 0 and i == -k):
-                D_squared = np.linalg.norm(T_B[i + k : i + k + m] - T_A[i : i + m]) ** 2
+                D_squared = (
```

Comment:
the variable name is now questionable 😅

Reply:
Indeed 😄

Reply:
@BBischof Any thoughts on what might be a more appropriate name?? `D_pow`??

Reply:
I think keeping it simple and using `diagonal_p_norm`. Below it's that sliding window so there may be some additional language that helps communicate its use.